Modules/Filter/include/selxDefaultComponents.h

```diff
@@ -44,6 +44,8 @@
 #include "selxItkResampleFilterComponent.h"
 #include "selxItkTransformSourceComponent.h"
 #include "selxItkTransformSinkComponent.h"
+#include "selxItkImageSourceFixedComponent.h"
```

Comment:
"selxItkImageSourceFixedComponent.h" and "selxItkImageSourceMovingComponent.h"
were delibrately removed. See 888e2ac , 83d27ec and fd5c790
See my message on that: https://github.com/SuperElastix/SuperElastix/commit/beddf56e65c4e4851fe67cd8190f12772acc5d5a#comments